### ISSUE TITLE
Accrued interest

### ISSUE THREAD

**Issue by hieronx** [embrio-tech/centrifuge-subql#234]

**Comment by filo87** [comment]
@hieronx, should that be tracked at single loan level? or aggregated at pool level? At what entity do you need this?

**Comment by hieronx** [comment]
@filo87 this was related to the P&L statement, so should be at the pool level. I imagine a `sumInterestAccruedByPeriod` property in `PoolSnapshot`

**Comment by filo87** [comment]
@hieronx Can we compute this as the variation in `outstandingInterest`from the portfolio call? What is there are Interest Repayments?

**Comment by hieronx** [comment]
@filo87 yeah I think it should be `outstandingInterest + repaidInterestAmountByPeriod - outstandingInterestPreviousPeriod`

Say the interest outstanding was previously 12 USDC, there is 1 USDC interest accrual per day, and there was an interest repayment of 5. Then new outstanding interest would be 8. We can derive interest accrued based on `5 + 8 - 12 = 1 USDC`.
